modules/ipi-install-preparing-to-deploy-with-virtual-media-on-the-baremetal-network.adoc

```diff
@@ -63,7 +63,7 @@ oc edit provisioning
 <1> Add `virtualMediaViaExternalNetwork: true` to the `provisioning` CR.
 
 
-. Edit the machineset to use the API VIP address:
+. If the image URL exist, edit the machineset to use the API VIP address. This step only applies to clusters installed in versions 4.9 or earlier.
```

Comment:
should machineset be in backticks?
```suggestion
. If the image URL exists, edit the `machineset` to use the API VIP address. This step only applies to clusters installed in versions 4.9 or earlier.
```

Reply:
Oh shoot interesting, yes its suppose to be in backticks, my preview shows it correctly but not the PR. Might have forgotten to push something. Thank you, nice catch!